### PR TITLE
makes extinguishers toggle being an open container based on safety, replaced the SAFETY = 0 and SAFETY = 1 with FALSE and TRUE

### PR DESCRIPTION
**Pull request by ZWRh3**
…eplaces the 0 and 1 with TRUE and FALSE

[XS] 
## About the PR 

Made fire extinguisher toggle between being an open/closed container based on the safety
this change has a side effect of allowing steam in fire extinguishers

## Why's this needed? 

i hated the fact that smoke powder could get out of an obviously closed container

## Changelog

```changelog
(u) ZWRh3
(+) Extinguishers with the safety on will no longer count as an open container.
```

### CODE REVIEW

**Comment by ZWRh3** [comment]
I hope i didnt fuck anything up since this is the first PR i have made ever.

**Comment by flappybatpal** [comment]
You don't need to add a label like [XS], that is automated.
I would say the labels for this is [BALANCE].
No real need to note that you change the 1 and 0 to TRUE and FALSE for such a small PR and simple change. Coders will easily understand why this is changed.
The changelog entry is accurate but probably not descriptive enough for players to actually understand what this does.

**Comment by ZWRh3** [comment]
edited the changelog

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/chemistry/tools/extinguisher.dm [hunk 1]
```diff
@@ -196,14 +196,16 @@
 		user.update_inhands()
 		src.desc = "The safety is off."
 		boutput(user, "The safety is off.")
-		safety = 0
+		ADD_FLAG(src.flags,OPENCONTAINER)
```
Comment (ZeWaka) [inline code/modules/chemistry/tools/extinguisher.dm:199]:
```suggestion
		ADD_FLAG(src.flags, OPENCONTAINER)
```

File: code/modules/chemistry/tools/extinguisher.dm [hunk 2]
```diff
@@ -196,14 +196,16 @@
...
+		safety = FALSE
 	else
 		src.item_state = "fireextinguisher0"
 		set_icon_state("fire_extinguisher0")
 		user.update_inhands()
 		src.desc = "The safety is on."
 		boutput(user, "The safety is on.")
-		safety = 1
+		REMOVE_FLAG(src.flags,OPENCONTAINER)
```
Comment (ZeWaka) [inline code/modules/chemistry/tools/extinguisher.dm:207]:
```suggestion
		REMOVE_FLAG(src.flags, OPENCONTAINER)
```

File: code/modules/chemistry/tools/extinguisher.dm [hunk 3]
```diff
@@ -196,14 +196,16 @@
...
+		safety = TRUE
 	return
```
Comment (ZeWaka) [inline code/modules/chemistry/tools/extinguisher.dm:209]:
```suggestion
```

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**